blazor/mention/getting-started.md

```diff
@@ -0,0 +1,351 @@
+
+{% tabs %}
+{% highlight razor tabtitle="~/Imports.razor" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.DropDowns
+
+{% endhighlight %}
+{% endtabs %}
+
+* Now, add the Syncfusion Mention component in razor file. Here, the Mention component is added in the **~/Pages/Index.razor** file under the **~/Pages** folder.
+
+{% tabs %}
+{% highlight razor %}
+
+<SfMention TItem="string">
+    <TargetComponent>
+        <div id="commentsMention" placeHolder="Type @@ and tag user" ></div>
+    </TargetComponent>
+</SfMention>
+
+<style>
+    #commentsMention {
+        min-height: 100px;
+        border: 1px solid #D7D7D7;
+        border-radius: 4px;
+        padding: 8px;
+        font-size: 14px;
+        width: 600px;
+    }
+    div#commentsMention[placeholder]:empty:before {
+        content: attr(placeholder);
+        color: #555;
+    }
+</style>
+
+{% endhighlight %}
+{% endtabs %}
+
+* Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the application. Then, the Syncfusion `Blazor Mention` component will be rendered in the default web browser.
+
+![Blazor Mention component](./images/blazor-mention-component.png)
+
+## Binding data source
+
+After initialization, To populate a Mention with data using the `DataSource` property
+
+{% tabs %}
+{% highlight razor %}
+
+<SfMention TItem="UserData" DataSource="@data">
+    <TargetComponent>
+        <div id="commentsMention" placeHolder="Type @@ and tag user" ></div>
+    </TargetComponent>
+    <ChildContent>
+        <MentionFieldSettings Text="Name"></MentionFieldSettings>
+    </ChildContent>
+</SfMention>
+
+<style>
+    #commentsMention {
+        min-height: 100px;
+        border: 1px solid #D7D7D7;
+        border-radius: 4px;
+        padding: 8px;
+        font-size: 14px;
+        width: 600px;
+    }
+    div#commentsMention[placeholder]:empty:before {
+        content: attr(placeholder);
+        color: #555;
+    }
+</style>
+
+@code {
+    public class UserData
+    {
+        public string Name { get; set; }
+    }
+    public List<UserData> data = new List<UserData>()
+    {
+        new UserData { Name = "Selma Rose" },
+        new UserData { Name = "Garth" },
+        new UserData { Name = "Robert" },
+        new UserData { Name = "William" },
+        new UserData { Name = "Joseph" },
+    };
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor Mention data binding](./images/blazor-mention-data-binding.png)
+
+## See Also
+
+* [Getting Started with Syncfusion Blazor for Client-Side in .NET Core CLI](../getting-started/blazor-webassembly-dotnet-cli)
+* [Getting Started with Syncfusion Blazor for Server-side in Visual Studio](../getting-started/blazor-server-side-visual-studio)
+* [Getting Started with Syncfusion Blazor for Server-Side in .NET Core CLI](../getting-started/blazor-server-side-dotnet-cli)
```

Comment:
![image](https://user-images.githubusercontent.com/29145454/208373433-3cb93054-d7fb-4c1c-9073-0dd3743d4b5f.png)
You have mapped the value in text field.